.github/workflows/deploy.yml

```diff
@@ -1,21 +1,13 @@
 name: deploy
 on:
+  pull_request: {}
   push:
-    branches:
-      - master
+    branches: [master]
 jobs:
   deploy:
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-20.04
     steps:
     - uses: actions/checkout@v2
-    - uses: actions/setup-python@v2
+    - uses: sidvishnoi/spec-prod@v1
       with:
-        python-version: 3.8
-    - run: pip install bikeshed
-    - run: bikeshed update
-    - run: bikeshed spec
-    - run: mkdir out && mv index.html out/
-    - uses: peaceiris/actions-gh-pages@v3
-      with:
-        github_token: ${{ secrets.GITHUB_TOKEN }}
-        publish_dir: out
+        GH_PAGES_BRANCH: gh-pages
```

Comment:
Is this really needed, doesn't this variable have a default value?

Reply:
This variable does two things: enables gh pages deployment as well as specify the branch. Not having a default value as not everyone wants to deploy built version to gh-pages

Reply:
I see, that makes sense!